guides/source/active_model_basics.md

```diff
@@ -550,40 +1316,40 @@ irb> person.valid?
 => false
 
 # When the confirmation doesn't match the password.
-irb> person.password = 'aditya'
-irb> person.password_confirmation = 'nomatch'
+irb> person.password = "aditya"
+irb> person.password_confirmation = "nomatch"
 irb> person.valid?
 => false
 
 # When the length of password exceeds 72.
-irb> person.password = person.password_confirmation = 'a' * 100
+irb> person.password = person.password_confirmation = "a" * 100
 irb> person.valid?
 => false
 
 # When only password is supplied with no password_confirmation.
-irb> person.password = 'aditya'
+irb> person.password = "aditya"
 irb> person.valid?
 => true
 
 # When all validations are passed.
-irb> person.password = person.password_confirmation = 'aditya'
+irb> person.password = person.password_confirmation = "aditya"
 irb> person.valid?
 => true
 
 irb> person.recovery_password = "42password"
 
-irb> person.authenticate('aditya')
+irb> person.authenticate("aditya")
```

Comment:
It might be worth adding a comment that `authenticate` is essentially a shortcut to `authenticate_password`. (first read through these examples might sound confusing, as to why there's two methods)